docs/notes/notes.rst

```diff
@@ -1,43 +1,38 @@
-Version 1.10.0
-==============
+Version 2.0.0
```

Comment:
The notes for the last version could already be in their own file which is then included here. If you do that, make sure to rename `previous_versions` directory to something like `versions`.

Reply:
I thought about that, but the title level is different for current and previous versions, so I left it this way.

Reply:
Sure, is not a big deal to create the file afterwards anyways